pkg/addons/data/Deployment.apps/default/registry-console.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   name: registry-console
   namespace: default
 spec:
-  replicas: 1
+  replicas: 2
```

Comment:
I don't think we care about the registry console but the web console.

Reply:
registry console is deprecated in 3.11 and won't be included in 4.0 (FYI)